Sources/Error Handling/ErrorUtils.swift

```diff
@@ -296,18 +296,32 @@ enum ErrorUtils {
         withUntypedError error: Error,
         fileName: String = #fileID, functionName: String = #function, line: UInt = #line
     ) -> PurchasesError {
+        func handlePublicError(_ error: PublicError) -> PurchasesError {
+            if let errorCode = ErrorCode(rawValue: error.code) {
+                return .init(error: errorCode, userInfo: error.userInfo)
```

Comment:
I might be wrong but when we convert this in `purchases-hybrid-common` https://github.com/RevenueCat/purchases-hybrid-common/blob/main/ios/PurchasesHybridCommon/PurchasesHybridCommon/ErrorContainer.swift#L23 the nsError.code code will be the backend's returned error code?

Reply:
This isn’t changing the code though.
This method is used because Swift doesn’t have typed throws, so we need a way to convert from `Error` to a typed `PurchasesError`. But this method won’t change the error at all. You can see that in the tests.